### ISSUE TITLE
Fatal SIGSEGV error soon after application start up

### ISSUE THREAD

**Issue by nickebbitt** [adoptium/adoptium-support#323]
## Summary

Shortly after application start up one replica for a service running 6 in total failed with the following:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f58b3e4cd68, pid=1, tid=57
#
# JRE version: OpenJDK Runtime Environment AdoptOpenJDK-11.0.11+9 (11.0.11+9) (build 11.0.11+9)
# Java VM: OpenJDK 64-Bit Server VM AdoptOpenJDK-11.0.11+9 (11.0.11+9, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# V  [libjvm.so+0x7fdd68]  G1ParCopyClosure<(G1Barrier)0, (G1Mark)0>::do_oop(unsigned int*)+0x58
#
# Core dump will be written. Default location: /core.%e.1.%t
#
# An error report file with more information is saved as:
# /atcloud/coredumps/java_hs_err_pid1.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/AdoptOpenJDK/openjdk-support/issues
#

[error occurred during error reporting (), id 0xb, SIGSEGV (0xb) at pc=0x00007f58b5e57e91]
```

## Steps to reproduce

Apologies, we've not been able to recreate the problem - just raising for visibility and to see if can be correlated with an existing known issue.

## Expected results

The JVM doesn't fail.

## Actual results

JVM crashed and process restarted. 

Core dump attached.

[java_hs_err_pid1.log](https://github.com/adoptium/adoptium-support/files/6822534/java_hs_err_pid1.log)

## Triaging info

Java version:

openjdk version "11.0.11" 2021-04-20
OpenJDK Runtime Environment AdoptOpenJDK-11.0.11+9 (build 11.0.11+9)
OpenJDK 64-Bit Server VM AdoptOpenJDK-11.0.11+9 (build 11.0.11+9, mixed mode)

What is your operating system and platform?

CentOS Linux 8

How did you install Java?

Via RPM e.g.

```
cat <<'EOF' > /etc/yum.repos.d/adoptopenjdk.repo
[AdoptOpenJDK]
name=AdoptOpenJDK
baseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/$releasever/$basearch
enabled=1
gpgcheck=1
gpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
EOF

dnf -y -q install adoptopenjdk-11-hotspot
```

Did it work before?

It worked ok on 5 of 6 replicas.

Did you test with other Java versions?

No

**Comment by karianna** [comment]
@nickebbitt Have you tried with an 11 nightily?

**Comment by nickebbitt** [comment]
@karianna the problem happened for a single replica of a service running 6 in total and we've not seen it since, also unable to recreate

so likely not a lot can be done, mainly just raised for visibility _just in case_ it's useful for anyone but understand that may not be the case

**Comment by karianna** [comment]
Several compiler crashes in this area were fixed 11.0.12-11.0.15 suspect this is resolved.
